src/oca_pre_commit_hooks/checks_odoo_module_po.py

```diff
@@ -262,40 +263,53 @@ def visit_entry(self, entry):
                 self.parse_printf(entry.msgid, entry.msgstr)
                 self.parse_format(entry.msgid, entry.msgstr)
             except PrintfStringParseError as str_parse_exc:
-                linenum = self._get_po_line_number(entry)
-                self.checks_errors["po-python-parse-printf"].append(
-                    f"{self.filename_short}:{linenum} "
-                    "Translation string couldn't be parsed "
-                    f"correctly using str%variables {str_parse_exc}"
-                )
+                if self.is_message_enabled("po-python-parse-printf"):
+                    linenum = self._get_po_line_number(entry)
+                    self.checks_errors["po-python-parse-printf"].append(
+                        f"{self.filename_short}:{linenum} "
+                        "Translation string couldn't be parsed "
+                        f"correctly using str%variables {str_parse_exc}"
+                    )
             except FormatStringParseError as str_parse_exc:
-                linenum = self._get_po_line_number(entry)
-                self.checks_errors["po-python-parse-format"].append(
-                    f"{self.filename_short}:{linenum} "
-                    "Translation string couldn't be parsed "
-                    f"correctly using str.format {str_parse_exc}"
-                )
+                if self.is_message_enabled("po-python-parse-format"):
+                    linenum = self._get_po_line_number(entry)
+                    self.checks_errors["po-python-parse-format"].append(
+                        f"{self.filename_short}:{linenum} "
+                        "Translation string couldn't be parsed "
+                        f"correctly using str.format {str_parse_exc}"
+                    )
+
+    @staticmethod
+    def iter_model_occurrences(entry: POEntry):
+        for occurrence in entry.occurrences:
+            if occurrence[0].startswith("model:"):
+                yield occurrence[0]
 
     def check_po(self):
         """* Check po-duplicate-message-definition (message-id)
         in all entries of PO files
 
-        We are not using `check_for_duplicates` parameter of polib.pofile method
```

Comment:
Why are you deleting this important comment?

Reply:
It explains internals of the method so I don't think it should appear in the docstring. Being in the docstring also meant it showed up in user docs (README) which I also think was not its place. 

I just moved it down below as a comment so that it no longer appears in the README.